homeassistant/components/cloud/__init__.py

```diff
@@ -5,13 +5,17 @@
 import logging
 import os
 
+import aiohttp
+import async_timeout
 import voluptuous as vol
 
 from homeassistant.const import (
     EVENT_HOMEASSISTANT_START, CONF_REGION, CONF_MODE)
 from homeassistant.helpers import entityfilter
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
 from homeassistant.util import dt as dt_util
-from homeassistant.components.alexa import smart_home
+from homeassistant.components.alexa import smart_home as alexa_sh
+from homeassistant.components.google_assistant import smart_home as ga_sh
```

Comment:
I think we should import this modules in comonent `__init__` files and load it with `hass.components.alexa.smart_home.async_message()`